src/main/scala/com/scylladb/migrator/readers/Cassandra.scala

```diff
@@ -126,7 +128,17 @@ object Cassandra {
 
       timestampsToFields
         .map {
-          case ((ttl, writetime), fields) =>
+          case ((ttl, writetime), fields) => {
+            val writetimestamp = writetime.getOrElse(CassandraOption.Unset)
+            val baseTTL = if (defaultTTL > 0) defaultTTL else 0L
```

Comment:
I wonder if it would be clearer to keep the `Option` in the `defaultTTL` parameter.

So, the parameter would have type `defaultTTL: Option[Long]`, and here you would write `val baseTTL = defaultTTL.getOrElse(0)`.